db/migrations.go

```diff
@@ -0,0 +1,42 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"embed"
+	"github.com/northmule/gophermart/internal/app/services/logger"
+	"github.com/pressly/goose/v3"
+)
+
+type Migrations struct {
+	mFS   embed.FS
+	sqlDB *sql.DB
+	ctx   context.Context
+}
+
+//go:embed migrations/*.sql
+var migrationsFS embed.FS
+
+func NewMigrations(db *sql.DB, ctx context.Context) *Migrations {
```

Comment:
Если конекст передается, как параметр стоит сделать его первым параметром.
```suggestion
func NewMigrations(ctx context.Context, db *sql.DB) *Migrations {
```